lib/std/http.zig

```diff
@@ -232,8 +231,7 @@ pub const Status = enum(u10) {
             200...299 => .success,
             300...399 => .redirect,
             400...499 => .client_error,
-            500...599 => .server_error,
-            else => .nonstandard,
+            else => .server_error,
```

Comment:
shouldn't this be: ?
```zig
500...599 => .server_error,
600...999 => .server_error,
else => unreachable,
```

to handle `< 100` and `>= 1000`

Reply:
all invalid status codes should be handled, making some unreachable will definitely cause massive issues when dealing with malicious servers.

Reply:
RFC 9110 section 15:
Values outside the range 100..599 are invalid. Implementations often use
three-digit integer values outside of that range (i.e., 600..999) for
internal communication of non-HTTP status (e.g., library errors). A
client that receives a response with an invalid status code **SHOULD
process the response as if it had a 5xx (Server Error) status code.**